packages/material-ui/src/Popper/Popper.js

```diff
@@ -173,6 +160,7 @@ class Popper extends React.Component {
     if (transition) {
       childProps.TransitionProps = {
         in: open,
+        onEntered: this.handleEntered,
```

Comment:
What do you think of using the enter event instead of entered?

Reply:
I think that makes more sense, aligns it _closer_ to what it was previously, and means it won't suddenly unmount and disappear whilst animating (entering).